compiler/x/codegen/X86Ops.ins

```diff
@@ -2797,6 +2797,26 @@ INSTRUCTION(MOV8RegReg, mov,
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM),
             PROPERTY1(IA32OpProp1_LongSource | IA32OpProp1_LongTarget),
             FEATURES(0)),
+INSTRUCTION(CMOVA4RegReg, cmova,
+            BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_0F__, 0x47, 0, ModRM_RM__, Immediate_0),
+            PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM | IA32OpProp_IntSource | IA32OpProp_IntTarget | IA32OpProp_TestsCarryFlag | IA32OpProp_TestsZeroFlag),
```

Comment:
@0xdaryl @kevindean12 Can we please review IA32OpProp properties more closely for the newly added X86Ops, especially - IA32OpProp_IntSource and IA32OpProp_IntTarget ?